v3/plugins/blues-man/quarkus-kotlin/0.1.0/meta.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+publisher: blues-man
+name: quarkus-kotlin
```

Comment:
This should match the publisher field in the [vscode-kotlin package.json](https://github.com/mathiasfrohlich/vscode-kotlin/blob/master/package.json), so in this case `vscode-kotlin`.